docs/02-help-desk/hardware-offerings.md

```diff
@@ -0,0 +1,91 @@
+
+# Requesting hardware or accessories
+
+## Services offered
+
+Bixal offers several different laptop configurations depending on role and job
+requirements. Beyond a laptop Bixal will provide a budget for accessories
+and offerers a list of accessories that can be requested without
+additional input from management.
+
+### A Note on Americans with Disabilities Act
+
+The equipment we purchase and distribute to employees works well for most
+employees. However Bixal understands that accommodations for health reasons
+should be made. If you feel like the standard equipment Bixal distributes
+will have a negative effect, or create challenges for you working at Bixal
+you should discuss that with People and Culture, your project manager,
+or create a ticket at [https://help.bixal.com](https://help.bixal.com)
+to request equipment that will allow you work and communicate with your
+peers without creating impedance.
+
+### Laptop Variants
+
+* Mac Book Pro 16"
+  * Insert specs here.
+* Mac Book Pro 14"
```

Comment:
The most current MacBook Pro for this tier is a 13". Are we referring to older models here?

Reply:
It *should* be the new one. I'm mistaken above